### ISSUE TITLE
Allow formatting in bibliographic entries

### ISSUE THREAD

**Issue by michael** [substance/texture#492]
Which tags should we allow?

- `em`
- `strong` (isn't it actually a bad practice to put strong=bold text in the titles?)
- `sub`
- `sup`

We should try to avoid formatting where possible. And I'd prefer semantic tags over `<i>` `<b>`...

**Comment by Melissa37** [comment]
Hi there

I agree strong should not be included - it's a display thing. However, people using mixed-citation might be presenting the formatting as well as presentation from within the XML?

Italics are essential, they are used to denote things like species, genes etc.

**Comment by fabiobatalha** [comment]
I agree with the comments.

Besides the tagged references we will have the ability to represent the original reference as described at #412. In this case we should preserve the basic formatting tags like `<b>`.

**Comment by dalapeyre** [comment]
I agree that italics are essential in mixed-citation, but in JATS, they might be achieved with 'named-content' element with an attribute for a genus-species type

**Comment by Melissa37** [comment]
Thanks @dalapeyre, but from a pragmatic point of view I am sure that might be tricky for publishers to implement with their production vendors as italics are not just for species names.

**Comment by dalapeyre** [comment]
Sorry I was unclear. That was my point. You said:
 'italics are not JUST for species names’

I agree, but further claim that Italics (<italic>)
is NOT for species-names at all.
 
 - Italics is for italics (<italic>), no reason stated.
 
 - <styled-content> is for any non-named emphasis type, such
   as 'I want to make this word “red"'. (The @style attribute
   does the same thing for table cells.)

 - <named-content> is for species names, using a type attribute,
   to mark this phrase semantically as a species name.

—Debbie


> On Sep 20, 2018, at 4:09 AM, Melissa Harrison <notifications@github.com> wrote:
> 
> Thanks @dalapeyre, but from a pragmatic point of view I am sure that might be tricky for publishers to implement with their production vendors as italics are not just for species names.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 


================================================================
Deborah A Lapeyre              mailto:dalapeyre@mulberrytech.com
Mulberry Technologies, Inc.      https://www.mulberrytech.com
17 West Jefferson Street         Phone: 301-315-9631 (USA)
Suite 207                        Fax:   301-315-8385
Rockville, MD 20850
----------------------------------------------------------------
Mulberry Technologies: Consultancy for XML, XSLT, and Schematron
================================================================

**Comment by michael** [comment]
Than you for your inputs. Regarding semantic tagging (e.g. species names), I think there's a lot of potential to improve the situation with tool support. We'll be working on that topic together with SourceData (EMBOPress) to tag concepts in JATS document. See here: https://github.com/substance/texture/issues/698
